### PR TITLE
canFind: added example to search for multiple needles in a hay stack.

### PR DESCRIPTION
**Pull request by carun**

### CODE REVIEW

File: std/algorithm/searching.d [hunk 1]
```diff
@@ -2555,6 +2555,18 @@ template canFind(alias pred="a == b")
     ];
     assert(!canFind(words, "bees"));
     assert( canFind!((string a, string b) => a.startsWith(b))(words, "bees"));
+
+}
+
+/// Search for mutliple items in an array of items (searching needles in a hay stack)
```
Comment (wilzbach) [inline std/algorithm/searching.d]: Typo: multiple
Also the description is wrong. You are searching for items in an array of array of items. Though that will be confusing to people.
Maybe something like:

Search for needles in an array of hay stacks

Comment (carun) [reply std/algorithm/searching.d]: @wilzbach I force pushed. Does the new changeset look OK?

File: std/algorithm/searching.d [hunk 2]
```diff
@@ -2557,6 +2557,17 @@ template canFind(alias pred="a == b")
     assert( canFind!((string a, string b) => a.startsWith(b))(words, "bees"));
 }
 
+/// Search for mutliple items in an array of items (search for needles in an array of hay stacks)
```
Comment (wilzbach) [inline std/algorithm/searching.d]: Still a typo here: multiple

Also it's probably a good idea to use multiple lines here as the soft limit is 80.

Comment (carun) [reply std/algorithm/searching.d]: Ah, I can't even spell multiple properly! I need to enable spell check "only" for comments.

Comment (wilzbach) [reply std/algorithm/searching.d]: Oh and sorry for just noticing now, haystack is one word: http://dictionary.cambridge.org/english/haystack

**Pull request merged**